unittests/AST/TensorFlow.cpp

```diff
@@ -99,3 +104,78 @@ TEST_F(TypeContainsTensorFlowValueTest, WorksForRecursiveTypes) {
   recursiveDecl->addMember(varDecl);
   EXPECT_FALSE(containsTensorFlowValue(recursiveType));
 }
+
+TEST_F(TypeContainsTensorFlowValueTest, TensorFlowType) {
+  EXPECT_TRUE(tf::isTensorHandle(tensorHandleType));
+  EXPECT_FALSE(tf::isTensorHandle(floatType));
+}
+
+TEST_F(TypeContainsTensorFlowValueTest, TensorHandleElementType) {
+  EXPECT_TRUE(tf::getTensorHandleElementType(floatType).isNull());
+  EXPECT_FALSE(tf::getTensorHandleElementType(tensorHandleType).isNull());
+}
+
+TEST_F(TypeContainsTensorFlowValueTest, ClassifyTensorFlowTypes) {
```

Comment:
Would be good to split this into separate tests.

Reply:
I did separate them on next commit.